modules/images-configuration-registry-mirror.adoc

```diff
@@ -0,0 +1,182 @@
+
+. Log into your {product-title} cluster.
+
+. Create an ImageContentSourcePolicy file (for example,
+`registryrepomirror.yaml`), replacing the source and
+mirrors with those of your own registry/repository pairs and images
+
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: ubi8repo
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - example.io/example/ubi-minimal<1>
+    source: registry.access.redhat.com/ubi8/ubi-minimal<2>
+  - mirrors:
+    - example.com/example/ubi-minimal
+    source: registry.access.redhat.com/ubi8/ubi-minimal
+~                                                 
+----
+<1> Indicates the name of the image registry/repository
+<2> Indicates the registry/repository containing the content being mirrored
+
+. Create the new ImageContentSourcePolicy:
++
+----
+$ oc create -f registryrepomirror.yaml
+----
++
+Once the ImageContentSourcePolicy has been created,
+the new settings will be deployed to each node
+and shortly start using the mirrored repository
+for requests to the source repository.
+
+. To check that the mirrored configuration worked,
+go to one of your nodes. For example:
++
+----
+$ oc get node
+NAME                           STATUS                     ROLES    AGE  VERSION
+ip-10-0-137-44.ec2.internal    Ready                      worker   7m   v1.14.6+90fadebfa
+ip-10-0-138-148.ec2.internal   Ready                      master   11m  v1.14.6+90fadebfa
+ip-10-0-139-122.ec2.internal   Ready                      master   11m  v1.14.6+90fadebfa
+ip-10-0-147-35.ec2.internal    Ready,SchedulingDisabled   worker   7m   v1.14.6+90fadebfa
+ip-10-0-153-12.ec2.internal    Ready                      worker   7m   v1.14.6+90fadebfa
+ip-10-0-154-10.ec2.internal    Ready                      master   11m  v1.14.6+90fadebfa
+----
++
+You can see that scheduling on each worker node is disabled as the change is being applied.
+
+. Check the `/etc/containers/registries.conf` file to make sure
+the changes were made:
++
+----
+$ oc debug node/ip-10-0-147-35.ec2.internal
+Starting pod/ip-10-0-147-35ec2internal-debug ...
+To use host binaries, run `chroot /host`
+
+sh-4.2# chroot /host
+sh-4.2# cat /etc/containers/registries
+unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
+[[registry]]
+  location = "registry.access.redhat.com/ubi8/"
+  insecure = false
+  blocked = false
+  mirror-by-digest-only = true
+  prefix = ""
+
+  [[registry.mirror]]
+    location = "example.io/example/ubi8-minimal"
+    insecure = false
+
+  [[registry.mirror]]
+    location = "example.com/example/ubi8-minimal"
+    insecure = false
+----
+
+. Pull an image to the node from the source and check if it is actually
+resolved by the mirror.
++
+----
+sh-4.2# podman pull --log-level=debug registry.access.redhat.com/ubi8/ubi-minimal
+----
+
+.Troubleshooting repository mirroring
+
+If the repository mirroring procedure doesn't work as described, here
```

Comment:
s/doesn't/does not
s/here is some/use the following

Reply:
Fixed.